src/js/components/CustomCardControl/CustomCardControl.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { FormattedMessage, injectIntl } from 'react-intl';
+import { CardControl } from 'cf-component-card';
+import { Button } from 'cf-component-button';
+import { CLOUDFLARE_UPGRADE_PAGE } from '../../constants/UrlPaths.js';
+
+class CustomCardControl extends Component {
+    static propTypes = {
+        name: PropTypes.string,
+    };
+
+    render() {
+        let { integrationName } = this.props;
+
+        // TODO: Add minimumPlan to lang folder 
+        let planList = {
+            "Free Website": 0,
+            "Pro Plan": 1,
+            "Biz Plan": 2,
+            "Ent Plan": 3
+        };
+
+        var currentPlan = this.props.hasOwnProperty("currentPlan") ? this.props.currentPlan : planList["Free Website"];
```

Comment:
In stead of passing this as a prop lets get it from zonesReducer.  Create a selectors/zones.js with getPlan(zoneId, state) which returns the `id` from that plan JSON with activeZoneId as the zoneId.